contents/blog/posthog-vs-kubit.md

```diff
@@ -0,0 +1,412 @@
+
+> ### What about product-minded engineers?
+>
+> Increasingly, many organizations are hiring engineers with a product-orientated approach, called _product engineers_. PostHog is especially helpful for these types of roles and we’ve written at length about [what product engineering is, and why it matters](/blog/what-is-a-product-engineer) — and [we even have a newsletter about it](/newsletter).
+
+In this article we’ll explore these differences in more detail, and take a deep dive into the product analytics features of both. 
+
+## How do PostHog and Kubit differ?
+
+### PostHog does more than product analytics
+Kubit is intended for product and data teams at large organizations. It therefore focuses solely on providing product analytics tools (and has an opaque pricing model that’s squarely aimed at enterprises), and that’s it.
+
+PostHog, on the other hand, offers far more than just product analytics and empowers engineers to collect feedback (with surveys), deploy changes (with feature flags), test ideas (with A/B testing) and gather more information (with session replays). In short, it’s just ridiculously more powerful. 
+
+### PostHog can gather multiple types of data
+Both Kubit and PostHog are built around the concept of _events_, which users define, track and analyze. An event can be performed, or not, and can have properties, or not. That’s about it. 
+
+However, what if you want to analyze something which isn’t an event, like a feedback comment, or how long a user spent reading a page? With Kubit that’s not possible, but with PostHog it is. You can watch replays of actual user interactions with your product, issue surveys to collect feedback — or even use PostHog as a data warehouse to track other data. 
+
+### PostHog is open source 
+At PostHog we aren’t just open source; we’re transparent by default. Everything from our [roadmap](/roadmap) to our [team profiles](/team) are available to the public. This means we’re constantly taking feedback on how to improve, as well as code contributions from our community that directly impact the product. Need an integration that doesn’t exist yet? You can build it yourself. 
+
+Kubit, on the other hand, is _incredibly_ closed source — so the point that pricing information is publicly available, free trials require personal information, and the roadmap is non-existent. That’s not the way we like to work.  
+
+## PostHog and Kubit feature comparison
+As explained above, PostHog offers a far greater suite of features than Kubit, which focuses solely on product analytics. In fact, PostHog’s suite of features is continually growing — check [the roadmap](/roadmap) to see what we’re working on next!
```

Comment:
```suggestion
As explained above, PostHog offers wider selection of feature than Kubit, which focuses solely on product analytics.
```

We've already mentioned the roadmap and this sounds a bit too heavy-handed to me.